packages/server/controllers/json/app.controller.js

```diff
@@ -2,8 +2,8 @@ import { getDataFromJson, groupBy } from './utils.js';
 
 export const all_apps = async (req, res) => {
 	const today = new Date();
-	const yesterday = new Date(today);
-	yesterday.setDate(yesterday.getDate() - 1);
+	const yesterday = new Date();
+	yesterday.setDate(today.getDate() - 1);
 
```

Comment:
This won't change anything. Results will be exactly the same. 

Whenever possible, try running the code and testing before pushing.

Reply:
Actually I changed it for my understanding, forgot to change it back